ZupIT/ritchie-cli
# Update Bucket env variable syntax

### Description

- Update Bucket env variable syntax

### How to verify it

- Check the workflow execution
---------------------------------------------### This pull request generated the following artifacts.
To test the health and quality of this implementation, download the respective binary for your operating system, unzip and directly run the binary like the examples below.
- **Windows** Download the file: **[rit-windows.zip](https://github.com/ZupIT/ritchie-cli/suites/3781046164/artifacts/93147638)** Unzip to some folder like: `C:\home\user\downloads\pr1041` Access the folder: `cd C:\home\user\downloads\pr1041` Directly call the binary: `.\rit.exe --version` or `.\rit.exe name of formula`  - **Linux** Download the file: **[rit-linux.zip](https://github.com/ZupIT/ritchie-cli/suites/3781046164/artifacts/93147636)** Unzip to some folder like: `/home/user/downloads/pr1041` Access the folder: `cd /home/user/downloads/pr1041` Assign execute permission to binary: `chmod +x ./rit` Directly call the binary: `./rit --version` or `./rit name of formula`  - **MacOS** Download the file: **[rit-macos.zip](https://github.com/ZupIT/ritchie-cli/suites/3781046164/artifacts/93147637)** Unzip to some folder like: `/home/user/downloads/pr1041` Access the folder: `cd /home/user/downloads/pr1041` Assign execute permission to binary: `chmod +x ./rit` Directly call the binary: `./rit --version` or `./rit name of formula`> Generated at Wed Sep 15 2021 21:04:30 GMT+0000 (Coordinated Universal Time)
